otx/api/usecases/exportable_code/visualizers/visualizer.py

```diff
@@ -111,3 +119,48 @@ def is_quit(self) -> bool:
             return False
 
         return ord("q") == cv2.waitKey(self.delay)
+
+    def save_frame(self, image, input_path, streamer_type) -> None:
```

Comment:
Could you add type hint?

Reply:
Added type hints to all functions I've added